src/Compilers/Core/Rebuild/Extensions.cs

```diff
@@ -2,12 +2,15 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
 using System.IO;
+using System.Linq;
 using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
 
 namespace BuildValidator
 {
-    internal static class BlobReaderExtensions
+    public static class Extensions
```

Comment:
>public [](start = 4, length = 6)

`internal`?

Reply:
This needs to be `public` because the method `GetEmbeddedPdbMetadataReader` is used by the `BuildValidator` project. The other method here does not need to be `public` though and I flipped that to `internal`.